.github/workflows/ci_with_docker_linux.yaml

```
@@ -12,9 +12,9 @@ jobs:
   docker-ci-standard:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
       - uses: docker/setup-buildx-action@v2
```

actionlint (1.7.12)

[error] the runner of "docker/setup-buildx-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)